specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -555,6 +555,183 @@
           }
         }
       }
+    },
+    "/weather/historical/actuals/daily/{format}": {
+      "get": {
+        "description": "**Get Daily Historical Actuals**\n  \n  \n**Applies to**: G1 and G2 pricing tiers.\n\n\nThe service returns climatology data such as past daily actuals for a given coordinate location.",
+        "operationId": "Weather_GetDailyHistoricalActuals",
+        "x-ms-examples": {
+          "DailyHistoricalActuals": {
```

Comment:
You may also consider friendly names for those example: like https://github.com/Azure/azure-rest-api-specs/blob/main/specification/maps/data-plane/Render/preview/2.1/render.json#L61